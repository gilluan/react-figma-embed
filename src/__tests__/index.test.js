import * as React from 'react';
import renderer from 'react-test-renderer';
import FigmaEmbed from '../index';

describe('FigmaEmbed', () => {
  it('renders the iframe correctly', () => {
    const tree = renderer
      .create(
        <FigmaEmbed url="https://www.figma.com/file/LKQ4FJ4bTnCSjedbRpk931/Sample-File" />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
