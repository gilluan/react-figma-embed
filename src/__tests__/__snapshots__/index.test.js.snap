// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FigmaEmbed renders the iframe correctly 1`] = `
<iframe
  allowFullScreen={true}
  height="450"
  src="https://www.figma.com/embed?embed_host=astra&url=https://www.figma.com/file/LKQ4FJ4bTnCSjedbRpk931/Sample-File"
  width="800"
/>
`;
